### ISSUE TITLE
[1.7.10] Crash when kill a mob

### ISSUE THREAD

**Issue by quentin452** [quentin452/1.7.10-Forge-renewed-v1#323]
```
java.lang.NullPointerException: Ticking memory connection
	at net.neverandy.fg.utility.EventHelper.onEntityKilled(EventHelper.java:60)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_1321_EventHelper_onEntityKilled_LivingDropsEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at net.minecraftforge.common.ForgeHooks.onLivingDrops(ForgeHooks.java:308)
	at net.minecraft.entity.EntityLivingBase.func_70645_a(EntityLivingBase.java:939)
	at net.minecraft.entity.EntityLivingBase.func_70097_a(EntityLivingBase.java:855)
	at net.minecraft.entity.passive.EntityAnimal.func_70097_a(SourceFile:145)
	at com.fiskmods.lightsabers.asm.ASMHooks.attackEntityFrom(ASMHooks.java:24)
	at net.minecraft.entity.player.EntityPlayer.func_71059_n(EntityPlayer.java:1232)
	at net.minecraft.network.NetHandlerPlayServer.func_147340_a(NetHandlerPlayServer.java:821)
	at net.minecraft.network.play.client.C02PacketUseEntity.func_148833_a(SourceFile:55)
	at net.minecraft.network.play.client.C02PacketUseEntity.func_148833_a(SourceFile:10)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.tclproject.mysteriumlib.asm.fixes.MysteriumPatchesFixesF.tick(MysteriumPatchesFixesF.java:51)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

**Comment by quentin452** [comment]
fixed in V0.28 by removing AdvancedLightsabers

**Comment by quentin452** [comment]
fixed by removing FlameGear-1.7.10-0.0.1 alpha in V0.29 **HOTFIX**
